### PR TITLE
correcting kyriad entry, wikidata now more specific

### LINKED ISSUE #1862: Research needed for: tourism/hotel|Kyriad
#### :thinking: We need help deciding what to do with "tourism/hotel|Kyriad"

:point_right: Read the [CONTRIBUTING.md](https://github.com/osmlab/name-suggestion-index/blob/master/CONTRIBUTING.md) guide for more info, and feel free to ask questions on this issue!

### What's needed

First, do some research on `tourism/hotel|Kyriad` by searching Google and Wikipedia.

#### If it is an actual brand:

* Edit `config/canonical.json`:
  * Find the entry for `tourism/hotel|Kyriad`
  * Add tags for `brand:wikidata` and `brand:wikipedia`
  * Optionally add `"countryCodes"` or other tags
* `npm run build`
* Commit your changes and submit a pull request

#### If it is not an actual brand:

* Edit `config/filters.json` to add an expression that discards it
* Edit `config/canonical.json` to remove the entry for `tourism/hotel|Kyriad`
* `npm run build`
* Commit your changes and submit a pull request

#### If the situation is complicated:

* Leave a comment on this issue with what you find

:sparkles: Thanks!

_This is an autogenerated ticket_

### PR DESCRIPTION
**Pull request by good-praxis**
fixes #1862 
The Kyriad hotels _do_ belong to the [Groupe du Louvre](https://en.wikipedia.org/wiki/Groupe_du_Louvre), however they do have their own [wikidata entry](https://www.wikidata.org/wiki/Q11751808) with a [corresponding wikipedia article](https://pl.wikipedia.org/wiki/Kyriad)

### CODE REVIEW

File: config/canonical.json [hunk 1]
```diff
@@ -33568,11 +33568,10 @@
   },
   "tourism/hotel|Kyriad": {
     "count": 76,
-    "countryCodes": ["fr"],
     "tags": {
       "brand": "Kyriad",
-      "brand:wikidata": "Q529212",
-      "brand:wikipedia": "fr:Louvre Hotels Group",
+      "brand:wikidata": "Q11751808",
+      "brand:wikipedia": "pl:Kyriad",
```
Comment (ENT8R) [inline config/canonical.json]: Why do you link to the Polish wikipedia page if this hotel does only exist in France?

Comment (ENT8R) [reply config/canonical.json]: See also https://overpass-turbo.eu/s/CWh

Comment (ENT8R) [reply config/canonical.json]: Mmhh.. that is some kind of suboptimal 😞 But as far as I can see, this hotel does only exist in France, so a Polish wikipedia article does not make that much sense... Maybe we should just remove it. What you you think @bhousel @matkoniecz ?

Comment (matkoniecz) [reply config/canonical.json]: IMHO linking weird language version is preferable over not linking wikipedia at all. For example some motorway crossings in Poland have German-language wikipedia links, as there is no Polish language article about them.

Comment (bhousel) [reply config/canonical.json]: > IMHO linking weird language version is preferable over not linking wikipedia at all. For example some motorway crossings in Poland have German-language wikipedia links, as there is no Polish language article about them.

yeah I agree with @matkoniecz - linking a weird wiki language page is better than linking no language at all 

In fact, the `brand:wikipedia` links are known to be pretty unstable.  However the `brand:wikidata` link is the much more stable id.

We can always go thru later and run a bulk edit on OpenStreetMap to cleanup the tags if better ones become available.  This is only possible once the editor start to assign the tags though - so yeah, imperfect tag is better than no tag 😄

**Pull request merged**